components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/verify/DigitalAssetLinksHandler.kt

```diff
@@ -79,12 +82,15 @@ internal class DigitalAssetLinksHandler(
             webDomain: String,
             packageName: String,
             fingerprint: String?,
-            relationship: String
+            relationship: String,
+            key: String?
         ) = URL.toUri().buildUpon()
             .appendQueryParameter(TARGET_ORIGIN_PARAM, webDomain)
             .appendQueryParameter(SOURCE_PACKAGE_NAME_PARAM, packageName)
             .appendQueryParameter(SOURCE_PACKAGE_FINGERPRINT_PARAM, fingerprint)
             .appendQueryParameter(RELATIONSHIP_PARAM, relationship)
+            .appendQueryParameter(PRETTY_PRINT_PARAM, false.toString())
+            .appendQueryParameter(KEY_PARAM, key)
```

Comment:
For debug builds, there wouldn't be a key passed, so this mean we send `null` as the value `key` since `appendQueryParameter` is silly.